build/php.m4

```diff
@@ -1929,6 +1929,20 @@ PKG_CHECK_MODULES([SQLITE], [sqlite3 >= 3.7.7], [
 ])
 ])
 
+dnl
+dnl PHP_SETUP_ZLIB(shared-add [, action-found, [action-not-found]])
+dnl
+dnl Common setup macro for zlib library. If zlib is not found on the system, the
+dnl default error by PKG_CHECK_MODULES is emitted, unless the action-not-found
+dnl is given.
+dnl
+AC_DEFUN([PHP_SETUP_ZLIB], [dnl
+PKG_CHECK_MODULES([ZLIB], [zlib >= 1.2.0.4], [dnl
```

Comment:
unrelated but 1.2.0.4 seems really old :)

Reply:
Released in 2003 :see_no_evil: oh my... yes, this can be also bumped.

Reply:
Interesting info is that pkg-config support was added in 1.2.3.1, so PHP already kind of requires 1.2.31 already :D

Reply:
And Windows zlib package is at 1.2.12 https://github.com/winlibs/zlib

Reply:
I've for now bumped this to 1.2.3.1 version... Probably could be bumped way more here.

Reply:
Note that centos 8 seems to use the 1.2.11 version, that might be a starting point as you re targetting master. Just my 2 cents :)

Reply:
Sounds good to me. Added in the next commit.